source/languages/en/riak/ops/running/authz.md

```diff
@@ -749,6 +748,37 @@ installing it can be found in [[Installing Erlang]]. This issue should
 not affect Erlang 17.0 and later.
 </div>
 
+## Enabling SSL
+
+In order to use any authentication or authorization features, you must
+enable SSL for Riak. **SSL is disabled by default**. If you are using
+[[Protocol Buffers|PBC API]] as a transport protocol for Riak (which we
+strongly recommend), enabling SSL on a given node requires only that you
+specify a [[host and port|Configuration Files#Client-Interfaces]] for
+the node as well as a [[certificate configuration|Authenticate and
+Authorization#Certificate-Configuration]].
+
+If, however, you are using the [[HTTP API]] for Riak and would like to
+configure HTTPS, you will need to not only establish a [[certificate
+configuration|Authentication and
+Authorization#Certificate-Configuration]] but also specify an HTTPS host
+and port. The following configuration would establish port 8088 on
+`localhost` as the HTTPS port:
+
+```riakconf
+listener.https.$name = 127.0.0.1:8088
+
+# By default, "internal" is used as the "name" setting
+```
+
+```appconfig
```

Comment:
@macintux The `app.config` settings are shown in all of the ops materials for the sake of people upgrading to 2.0 from 1.4.x